test/runtests.bat

```diff
@@ -7,7 +7,11 @@ REM compile the code into the bin folder
 javac  ..\src\seedu\addressbook\Addressbook.java -d ..\bin
 
 REM run the program, feed commands from input.txt file and redirect the output to the actual.txt
-java -classpath ..\bin seedu.addressbook.AddressBook < input.txt > actual.txt
+java -classpath ..\bin seedu.addressbook.AddressBook %" " > actual.txt
```

Comment:
Oh I should've removed the `%`, it was used due to my confusion with windows batch scripting. I will try adding the `< NUL`.